hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -99,6 +105,44 @@ def call(self, **kwargs) -> Response:
         return delete(self.url)
 
 
+@dataclass
+class ReactActionPerformer(ActionPerformer):
```

Comment:
In my mind, the concept of models is supposed to be along the lines of entities, or the definition of the types / shapes of objects. But this is processing (and the calling of webhooks) are not really entity / type -related at all. What about having this and the subclasses live in a separate module?

Reply:
I'm good moving this to a separate module